CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+The following is a set of guidelines for contributing to `si4ti`.
+
+Some of the ways you can help:
+
+- Submitting bug reports via [GitHub
+  Issues](https://github.com/equinor/si4ti/issues) of the repository. For
+  security related bugs, refer to the information in
+  [SECURITY.md](https://github.com/equinor/si4ti/blob/main/SECURITY.md).
+- Proposing code for bug fixes and new features, then [making a pull
+  request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
+- Fixing typos and generally improving the documentation.
+- Writing tutorials, examples, and how-to documents.
+
+## Pull request process
+
+1. Work on your own fork of the main repo.
+1. Push your commits and make a draft pull request.
+1. Check that your pull request passes all tests.
+1. When all tests have passed and you are happy with your changes, change your
+pull request to "ready for review", and ask for a code review.
+
+### Commits
+
+We strive to keep a consistent and clean git history and all contributions
+should adhere to the following:
+
+1. All tests should pass on all commits
+1. A commit should do one atomic change on the repository
+1. The commit headline should be descriptive and in the imperative.
+1. Commit description should be added to commits which require context
+   (explanation why the commit was introduced, reasoning for taken decisions,
+   important information about commit consequences, etc).
+
+### Testing
+
+Add tests of appropriate level for all new functionality/fixed bugs.
+
+### Style
+
+Style code similarly to neighboring code. Apply `pre-commit` for if applicable.
```

Comment:
There is no `pre-commit-config` file to use, so I don't think we want this line in the repo.

Reply:
As discussed, I do not plan to actually introduce `pre-commit` to the repository for now so I removed this passage from the file.